src/Xamarin.Android.Build.Tasks/Utilities/MamJsonParser.cs

```diff
@@ -57,17 +57,20 @@ public XElement ToXml ()
 				GetReplacementMethods ());
 		}
 
-		static JObject ReadJson (string path)
+		static JsonObject ReadJson (string path)
 		{
-			using (var f = File.OpenText (path))
-			using (var r = new JsonTextReader (f))
-				return (JObject) JToken.ReadFrom (r);
+			using (var f = File.OpenRead (path)) {
+				return JsonNode.Parse (f)!.AsObject ();
```

Comment:
This one I wonder if you should use `?` and then throw.

But maybe that would only fail on a document that has the text `null` inside on one line.